### ISSUE TITLE
Known linter issues

### ISSUE THREAD

**Issue by timfelle** [ExtremeFLOW/neko#1405]
- [ ] Double spaces should be allowed inside strings.
- [ ] Comment spacing rule triggered in strings ended with line breaks.

**Comment by MartinKarp** [comment]
It might just be me, maybe I installed the wrong version, but the options of flint that I have locally are: `config, dump, score, and tree`.

So when I run `flint lint` I get an error.

My flinter is located in:
```
$ whereis flint
flint: /home/martin/anaconda3/bin/flint
```
and my version should be
```
$ conda list flint
# packages in environment at /home/martin/anaconda3:
#
# Name                    Version                   Build  Channel
flinter                   0.4.0                    pypi_0    pypi
```

**Comment by timfelle** [comment]
Yes, if you look at the CI you will see that we actually use 2 different versions of flint.

The latest Flint crashes with some of the complex recursive structures in the `stack` and `htable`. Therefore we use version 0.4.0 to `score` the entire repository.

We use the latest Flint to `lint` all the changed files, since the output is a lot more readable than `0.4.0`.

If you upgrade your flint then the `lint` command will work. Otherwise I seem to recall that you need to use `dump` for `0.4.0`, since score only works for directories I think.

**Comment by MartinKarp** [comment]
Thanks! How should we do with the docs you think? Right now its 0.4.0, but lint would be more helpful for contributors.
